react/src/components/SessionDetailContent.tsx

```diff
@@ -193,25 +206,44 @@ const SessionDetailContent: React.FC<{
           )}
         </Descriptions.Item>
         <Descriptions.Item label={t('session.launcher.MountedFolders')}>
-          {_.map(
-            _.zip(legacy_session?.mounts, session?.vfolder_mounts),
-            (mountInfo) => {
-              const [name, id] = mountInfo;
-              return (
-                <Button
-                  key={id}
-                  type="link"
-                  size="small"
-                  icon={<FolderOutlined />}
-                  onClick={() => {
-                    open(id ?? '');
-                  }}
-                >
-                  {name}
-                </Button>
-              );
-            },
-          )}
+          {baiClient.supports('vfolder-nodes')
```

Comment:
Just check it using `session.vfolder_nodes`. If it support it should be not undefined.
```suggestion
          {session.vfolder_nodes !== undefined
```